net/sslh/src/opnsense/mvc/app/controllers/OPNsense/Sslh/forms/settings.xml

```diff
@@ -0,0 +1,197 @@
+<form>
+    <field>
+        <id>settings.enabled</id>
+        <label>Enable SSLH</label>
+        <type>checkbox</type>
+        <help><![CDATA[
+            This will enable the SSLH service.
+            ]]>
+        </help>
```

Comment:
```suggestion
        <help>This will enable the SSLH service.</help>
```

where possible this short inline style would be the best approach for translation work and maintenance